__std__/plutus-apps/packages/plutus-playground-client-entrypoint.nix

```diff
@@ -0,0 +1,41 @@
+{ inputs, cell }:
```

Comment:
Is this dead? Given that plutus-playground-client isn't a thing any more

Reply:
I'm not sure, cos this seems to include the docs as well. @koslambrou?

Reply:
Yeah, the haddocks are still hosted on the playground url: https://playground.plutus.iohkdev.io/doc/haddock/

This can be deleted once we switch to Github pages.